packages/lib/src/vis/utils.ts

```diff
@@ -348,7 +345,8 @@ export function getUniforms(
 }
 
 export function toArray(arr: NumArray): number[] {
-  return isTypedArray(arr) ? [...arr] : arr;
+  // eslint-disable-next-line unicorn/prefer-spread
+  return isTypedArray(arr) ? Array.from(arr) : arr;
```

Comment:
I thought it may be limited to a maximum number of arguments like when used in a function call (e.g. `Math.max(...arr)`) but it seems to be parsed and interpreted differently. I will revert.